### ISSUE TITLE
Warp Scroll is bugged in San d'Orian Consulate @ Windurst

### ISSUE THREAD

**Issue by Kai-Tybalt** [EdenServer/community#1637]
### Checklist

- [x ] I have searched for existing issues for issues like this one. The issue has not been posted. (Duplicate reports slow down development.)
- [x ] I have provided reproducable steps. (No "as the title says" posts please. Provide reproducable steps even if it seems like a waste of time.)
- [x ] I have provided my client version in the details. (type /ver into your game window)

### Details
![Screenshot (7)](https://user-images.githubusercontent.com/43795746/57585186-6f70ea00-7499-11e9-900f-7b73202985ca.png)
![Screenshot (8)](https://user-images.githubusercontent.com/43795746/57585187-6f70ea00-7499-11e9-8930-89783afc587c.png)

**Version:** 30180904_0
When purchasing a Warp Scroll from the San d'Orian Consulate in Windurst, the dialogue refers to gil cost and then seems to deduct about half of the character's current Conquest Points Total. I have purchased this item three times on separate days with the same effect.

I've attached pics for text reference. Also sorry for the white squares in the pics. My screenshot wouldn't work on ff xi for some reason so I was technically taking screenshots of document files with the game in the background.

**Comment by godmode** [comment]
Eden does not support this client version. Please check tech-support on Discord if you are having trouble installing. Also please read up on server information regarding costs.

**Comment by Kai-Tybalt** [comment]
I have updated to current version: 30181205_0
After following your instructions to update client, the bug persists. The Scroll of Instant Warp costs about 50% of current CP total. I've attached images of the chat log.

Thank you for your time! - An eternally grateful player
![Instant Warp Bug (1)](https://user-images.githubusercontent.com/43795746/57892890-9d399400-77f5-11e9-8400-13e697b3ffce.png)
![Instant Warp Bug (2)](https://user-images.githubusercontent.com/43795746/57892891-9dd22a80-77f5-11e9-988e-0be6379f7afc.png)
